src/Contrib/Service/Masa.Contrib.Service.MinimalAPIs/Attributes/RoutePatternAttribute .cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc;
+
+[AttributeUsage(AttributeTargets.Method)]
+public class RoutePatternAttribute : Attribute
+{
+    public string Pattern { get; set; }
+
+    public string? HttpMethod { get; set; }
```

Comment:
string? -> HttpMethod

Reply:
the default is null (the request method is automatically identified according to the method name prefix)

Reply:
unless custom request method